### PR TITLE
refactor: update int tests to move init checks to create

### PR DESCRIPTION
**Pull request by c-ryan-k**
---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

File: azext_edge/tests/edge/init/int/test_init_int.py [hunk 1]
```diff
@@ -44,17 +44,16 @@ def init_test_setup(settings, tracked_resources):
...
-        f"--rn {registry_namespace} --sa-resource-id {storage_account['id']} "
+        f"--rn {registry_namespace} --sa-resource-id {storage_account['id']}"
     )
     tracked_resources.append(registry["id"])
 
     if not all([settings.env.azext_edge_cluster, settings.env.azext_edge_rg]):
         raise AssertionError(
-            "Cannot run init tests without a connected cluster and resource group. "
-            f"Current settings:\n {settings}"
+            "Cannot run init tests without a connected cluster and resource group. " f"Current settings:\n {settings}"
```
Comment (vilit1) [inline azext_edge/tests/edge/init/int/test_init_int.py:56]: if you are going to do this then at least combine the strings...

File: azext_edge/tests/edge/init/int/test_init_int.py [hunk 2]
```diff
@@ -92,26 +89,24 @@ def test_init_scenario(
...
+    )
     # TODO: assert create when return be returning
     run(create_command)
 
     if init_test_setup["redeployment"]:
-        run(
-            f"az iot ops delete --name {instance_name} -g {resource_group} "
-            "-y --no-progress --force"
-        )
+        run(f"az iot ops delete --name {instance_name} -g {resource_group} " "-y --no-progress --force")
```
Comment (vilit1) [inline azext_edge/tests/edge/init/int/test_init_int.py:109]: look at that space

File: azext_edge/tests/edge/init/int/test_init_int.py [hunk 3]
```diff
@@ -92,26 +89,24 @@ def test_init_scenario(
     resource_group = init_test_setup["resourceGroup"]
     registry_id = init_test_setup["schemaRegistryId"]
     instance_name = init_test_setup["instanceName"]
-    command = f"az iot ops init -g {resource_group} --cluster {cluster_name} "\
-        f"--no-progress {additional_init_args} "
+    command = f"az iot ops init -g {resource_group} --cluster {cluster_name} " f"--no-progress {additional_init_args} "
```
Comment (vilit1) [inline azext_edge/tests/edge/init/int/test_init_int.py:92]: space

**Review by vilit1 — APPROVED** [review]
space

File: azext_edge/tests/edge/init/int/test_init_int.py (hunk 1 above)
Comment (c-ryan-k) [reply azext_edge/tests/edge/init/int/test_init_int.py:56]: sorry i didn't notice - that was black auto-formatting, will adjust some of these

**Pull request merged**